corehq/toggles.py

```diff
@@ -477,7 +477,7 @@ def toggle_values_by_name(username=None, domain=None):
     'user_reports',
     'User configurable reports UI',
     TAG_SOLUTIONS,
-    [NAMESPACE_DOMAIN, NAMESPACE_USER],
+    [NAMESPACE_DOMAIN],
```

Comment:
@sheelio Why did you change this? I think it's going to make it much harder for us to debug UCRs if we can't go look at them

Reply:
If the project is in the feature flag, you would still have access right?

Reply:
I'm generally pro domain namespace only, but I think this can also be helpful to debug report builder reports, and being able to rebuild tables manually on any domain.

If we give the access to the domain, then the "edit report" button would show up for all users in that domain (likely bad for enikshay/icds right?)